src/client/Microsoft.Identity.Client/Http/HttpManagerWithRetry.cs

```diff
@@ -84,9 +97,30 @@ internal class HttpManagerWithRetry : HttpManager
             return await SendRequestAsync(uri, headers, body, HttpMethod.Post, logger, retry: true, doNotThrow: true, cancellationToken: cancellationToken).ConfigureAwait(false);
         }
 
-        protected override HttpClient GetHttpClient()
+        public override async Task<HttpResponse> SendPostForceResponseAsync(
+            Uri uri,
+            IDictionary<string, string> headers,
+            StringContent body,
+            X509Certificate2 bindingCert,
+            ILoggerAdapter logger,
+            CancellationToken cancellationToken = default)
+        {
+            return await SendRequestAsync(uri, headers, body, HttpMethod.Post, logger, bindingCert, doNotThrow: true, cancellationToken: cancellationToken).ConfigureAwait(false);
+        }
+
+        protected override HttpClient GetHttpClient(X509Certificate2 x509Certificate2)
```

Comment:
This impl seems identical to the virtual method in the base class. Why did you override it?